lib/modules/manager/gomod/readme.md

```diff
@@ -22,3 +24,19 @@ Indirect updates are disabled by default. To enable them, add a package rule suc
   ]
 }
 ```
+
+### Private Modules Authentication
+
+Before executing `go` commands to update the `go.sum`, Renovate will export `git` [`insteadOf`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtinsteadOf) directives in environment variables.
+
+The following logic is executed prior to "artifacts" updating:
+
+The token from the `hostRules` entry matching `hostType=github` and `matchHost=api.github.com` is added as the default authentication for `github.com`.
+For those running against `github.com`, this will be the default platform token.
```

Comment:
For me it's not clear at a glance what _this_ points at. Maybe you can explain what _this_ is?